.changeset/honest-students-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+"@justeattakeaway/pie-notification": patch
```

Comment:
Super small note (or NIT 😆 ) – patch releases should only be used for tiny bug fixes or one-line updates, rather than added features (which should be minor releases).

Reply:
@ashleynolan noted, thanks for the feedback